src/components/Dropdown/_Dropdown.scss

```diff
@@ -0,0 +1,80 @@
+.dropdown {
+  color: inherit;
+  display: inline-block;
+
+  &__content {
+    background-color: var(--ui-white);
+    border: 1px solid var(--ui-gray-medium);
+    display: none;
+    padding: var(--space-xxs) var(--space-s) var(--space-xs);
+    position: absolute;
+    transition: all 0.35s;
+    width: 259px;
+    z-index: 1;
+  }
+
+  &__label {
+    @include button--outline;
+
+    align-items: center;
+    border: 1px solid var(--ui-gray-medium);
+    cursor: pointer;
+    font-weight: 300;
+    justify-content: space-between;
+    width: 148px;
```

Comment:
Please remove the defined width—we'll want this to explain for whatever string we populate the dropdown with